### DISCUSSION TITLE
✅ Allow YouTube Shorts in Video Widget

### DISCUSSION THREAD

**Discussion by planforawesome** [elementor/elementor#24220] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I want to embed YouTube Shorts in my blog posts, but when I insert the URL, the entire widget goes blank.

### Describe the solution you'd like

It would be great if YouTube Shorts functioned just like normal YouTube videos do.
Put the URL in, and it plays.

### Describe alternatives you've considered

Maybe a separate widget for YouTube Shorts if it's too complex to add them to the existing video widget?

**Comment by GoldfinchM** [comment]
YES, would love this to be a thing. Trying to add a media carousel for a client, who has a number of short videos which YouTube insist are 'shorts' but then I can't link them in the website :(

**Comment by nicholaszein** [comment]
This request was also submitted as the following issues:

- #20330 
- #28720

Which are on the same subject. All comments and reactions from those issues will be considered if this request gets enough votes.

Kind regards

**Comment by nicholaszein** [comment]
Hi there!

We have great news! 🙌 

### 📢 We're happy to announce that YouTube Shorts support for the Video widget was released with Elementor Core v3.27.0! 🥳 

You can now share YouTube shorts in the **Video widget**, and even change the aspect ratio. Just select the **`Source`** as **`YouTube`**, and paste your YouTube shorts link. Then in the **`Style`** tab you can change the Aspect Ration to 9:16.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
